cdap-docs/admin-manual/source/security/impersonation.rst

```diff
@@ -4,10 +4,42 @@
 
 .. _admin-impersonation:
 
-=========================
-Configuring Impersonation
-=========================
+=============
+Impersonation
+=============
 
-Impersonation allows users to run programs and access datasets, streams, and other resources as pre-configured users.
-Currently CDAP supports configuring impersonation at a namespace level, which means that every namespace will have
-a single user that programs in that namespace will run as, and that resources will be accessed as.
+Impersonation allows users to run programs and access datasets, streams, and other
+resources as pre-configured users. Currently CDAP supports configuring impersonation
+at a namespace level, which means that every namespace will have a single user that
+programs in that namespace will run as, and that resources will be accessed as.
+
+Requirements
+============
+To utilize this feature, Kerberos must be enabled on the cluster and configured in
+:ref:`cdap-site.xml <appendix-cdap-site.xml>`, via the parameter ``kerberos.auth.enabled``.
+To configure a namespace to have impersonation, specify the ``principal`` and
```

Comment:
added 'Kerberos' to a few of the instances, but not this one. It seemed strange to have it here.
See the updated changes:
https://github.com/caskdata/cdap/pull/6556/commits/d57759bd90b1d0a2e1697c109ab5837f7a36f948